fast-af/fast-checkout-woocommerce
# PINT-450 - Add page redirect option

# Description

Add an option under Fast settings to allow the admin to choose where the store redirects after successful checkout.

# Testing instructions

1. Login to [staging admin](fasttestdev.wpcomstaging.com/wp-admin/).
2. [Visit the Fast settings Options tab](https://fasttestdev.wpcomstaging.com/wp-admin/admin.php?page=fast&tab=fast_options).
3. For "Checkout Redirect Page", select any page. To select a page, start typing the name of the page, and it should appear so that you can select it.
4. Click the Save Changes button.
5. Visit https://fasttestdev.wpcomstaging.com/?fast_order_created=1 and verify that it redirects to the page you selected.

# Checks

@ilkerulutas @brikr this is ready for review.
